### DISCUSSION TITLE
Cannot run playbook

### DISCUSSION THREAD

**Discussion by Anjumvj** [learndevopsonline/learndevopsonline#1652] [Q&A] [answered]
![Screenshot 2023-12-02 101339](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/6c804014-a00f-4384-a5a0-3907857e65f0)

**Comment by Anjumvj** [comment]
![Uploading Screenshot 2023-12-02 101932.png…]()

**Reply by sreedharm07** [reply]
@Anjumvj 
Your sample.yml file is not following indentation(spaces) in ansible indentation is imp 
Please modify that and try 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/96b61800-d089-4636-a6c9-de9ce887f553)

**Comment by Anjumvj** [comment]
so do i have to give sample .yml?? after sample space then .yml?

**Reply by sreedharm07** [reply]
please use only one thread dont use multiple 

space means not sample .yml 
inside sample.yml content is not folowing indentation that meass spaces and sequenses line by ine 

but you written -

-name: sample
hosts: all
tasks:
       -name: ping
       ansible.builtin.ping:

actual file shoul be -
   - name: sample
     hosts: all
     tasks:
       - name: ping
         ansible.builtin.ping:

compare the actual file and your file and 
please gothrough ansible basic session again for better understanding

**Reply by Anjumvj** [reply]
okok got it. thanks a lot

**Comment by Anjumvj** [comment]
![Screenshot 2023-12-02 112525](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/dffd4a04-5a61-462a-9ba6-6efbfebc04ee)
![Screenshot 2023-12-02 112607](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/8729c6b0-03bb-439f-8112-960fd734aeb4)

**Reply by sreedharm07** [reply, marked as answer]
Again same
No indication followed 
- name: sample
  Hosts: all

Please go through ansible 1 st session again 

You will understood clearly

**Reply by Anjumvj** [reply]
ok
